CHANGELOG.md

```diff
@@ -1,3 +1,4 @@
+- Added uyuni-storage-setup-[server|proxy] packages to Server/Proxy deployment
```

Comment:
I am not sure if this should be:
```suggestion
- Added uyuni-storage-setup-server package to Server deployment and
  uyuni-storage-setup-proxy to Proxy deployment
```
But I will leave it to @keichwa to decide.

Reply:
Yes, I agree this should be changed, but will wait for Karl before committing the suggestion.

Reply:
suggestions sounds good.  Added to batch commission.  If you are faster, also fine.